platform/migration/platform-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -76,12 +85,37 @@ protected Object doExecute() {
                 for (MigrationWarning migrationWarning : migrationWarnings) {
                     outputWarningMessage(migrationWarning.getMessage());
                 }
+
                 outputWarningMessage(String.format(FAILED_EXPORT_MESSAGE, exportDirectory));
             }
         } catch (Exception e) {
             outputErrorMessage(String.format(ERROR_EXPORT_MESSAGE, e.getMessage()));
         }
+
         return null;
     }
 
+    private Subject getSubject() {
```

Comment:
While we're in here refactoring, this method seems like a better candidate for the Security utilities as well. I don't think this is the only command that we want to fall back to the system user, is it?